bindings/java/iota-client-java/src/main/java/org/iota/apis/MiscellaneousApi.java

```diff
@@ -0,0 +1,134 @@
+
+    public float getMinPoWScore() throws ClientException {
+        Float responsePayload = callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "GetMinPoWScore")).getAsFloat();
+        return responsePayload;
+    }
+
+    public int getTipsInterval() throws ClientException {
+        Integer responsePayload = callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "GetTipsInterval")).getAsInt();
+        return responsePayload;
+    }
+
+    public boolean getLocalPow() throws ClientException {
+        Boolean responsePayload = callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "GetLocalPoW")).getAsBoolean();
+        return responsePayload;
+    }
+
+    public boolean isFallbackToLocalPoW() throws ClientException {
+        Boolean responsePayload = callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "GetFallbackToLocalPoW")).getAsBoolean();
+        return responsePayload;
+    }
+
+    public Node[] getUnsyncedNodes() throws ClientException {
+        JsonArray responsePayload = (JsonArray) callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "UnsyncedNodes"));
+
+        Node[] nodes = new Node[responsePayload.size()];
+        for (int i = 0; i < responsePayload.size(); i++) {
+            nodes[i] = new Node(responsePayload.get(i).getAsJsonObject());
+        }
+
+        return nodes;
+    }
+
+    public PreparedTransactionData prepareTransaction(SecretManager secretManager, GenerateAddressesOptions generateAddressesOptions) throws ClientException {
+        JsonObject o = new JsonObject();
+        o.add("secretManager", secretManager.getJson());
+        o.add("generateAddressesOptions", generateAddressesOptions.getJson());
+
+        JsonObject responsePayload = (JsonObject) callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "PrepareTransaction", o));
+
+        return new PreparedTransactionData(responsePayload);
+    }
+
+    public TransactionPayload signTransaction(SecretManager secretManager, PreparedTransactionData preparedTransactionData) throws ClientException {
+        JsonObject o = new JsonObject();
+        o.add("secretManager", secretManager.getJson());
+        o.add("preparedTransactionData", preparedTransactionData.getJson());
+
+        JsonObject responsePayload = (JsonObject) callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "SignTransaction", o));
+
+        return new TransactionPayload(responsePayload);
+    }
+
+    public void storeMnemonic(SecretManager secretManager, String mnemonic) throws ClientException {
+        JsonObject o = new JsonObject();
+        o.add("secretManager", secretManager.getJson());
+        o.addProperty("mnemonic", mnemonic);
+
+        callBaseApi(new ClientCommand(ClientCommand.CommandType.CallClientMethod, "StoreMnemonic", o));
+    }
+
+    public Block postBlockPayload(BlockPayload payload) throws ClientException {
```

Comment:
Not aligned with the message_interface, I think here renaming the part in the message interface would also be fine